src/index.ts

```diff
@@ -268,10 +268,12 @@ function teenyRequest(
     options.compress = false;
     fetch(uri, options).then(
       res => {
-        res.body.on('error', err => {
-          console.log('whoa there was an error, passing it on: ' + err);
-          requestStream.emit('error', err);
-        });
+        res.body
+          .on('error', err => {
+            console.log('whoa there was an error, passing it on: ' + err);
```

Comment:
Yikes!  That `console.log` shouldn't be there.  Can you remove this while you're in there?

Reply:
I totally can! Though I am noticing there are 3 different logs in the code, [1 of which appears to happen in lieu of error handling](https://github.com/googleapis/teeny-request/blob/master/src/index.ts#L217) 😱 We might just want to open a separate PR to remove all them and implement error handling?